### ISSUE TITLE
`vue/no-v-text`

### ISSUE THREAD

**Issue by inker** [vuejs/eslint-plugin-vue#1435]
**Please describe what the rule should do:**

This rule would report usage of `v-text`.

**What category should the rule belong to?**

[X] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<template>
  <div v-text="foobar" />
</template>
```

**Additional context**

This issue can be fixable. The above code would be fixed to:
```vue
<template>
  <div>
    {{foobar}}
  </div>
</template>
```
